static/src/js/components/Button/Button.js

```diff
@@ -82,6 +83,7 @@ export const Button = ({
       aria-label={label}
       type={type}
       disabled={disabled || spinner}
+      target={target}
```

Comment:
Looks like `react-bootstrap` doesn't do anything to account for the issues with `target="_blank"` I think we can just add the `rel="noopener nofollow"` when `target === "_blank"` or something similar.

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/